c/openbsd-6.2/sources/007_etherip.patch.sig

```diff
@@ -0,0 +1,36 @@
+untrusted comment: signature from openbsd 6.2 base secret key
```

Comment:
What is a `.patch.sig` file? Is it intended to actually exist?

And is this first line also intended? It looks like an oversight?

Reply:
This is exactly the file that was provided online at the [errata page](https://www.openbsd.org/errata62.html) ([file](https://ftp.openbsd.org/pub/OpenBSD/patches/6.2/common/007_etherip.patch.sig)).
As stated:

> Each patch is cryptographically signed with the signify(1) tool and contains usage instructions

Documentation is available at the [manual page](http://man.openbsd.org/signify).
The file is used with the [patch program](http://man.openbsd.org/patch) in the Makefile to generate a fixed and a unfixed program file. 

I can remove the comment if you like but patch usually just ignores everything it can’t apply. 
I will mention the file in the Readme in the next commit.